src/main/java/org/myrobotlab/service/AudioFile.java

```diff
@@ -508,6 +530,7 @@ public ServiceConfig getConfig() {
     config.currentPlaylist = currentPlaylist;
     config.volume = volume;
     config.playlists = playlists;
+    config.peakMultiplier = peakMultiplier ;

```

Comment:
we should add peakSampleInterval here also.

Reply:
nice ! :+1: